.github/renovate.json5

```diff
@@ -0,0 +1,4 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["config:recommended", ":maintainLockFilesWeekly"],
```

Comment:
Does this mean that we would get a new PR every week? I'd prefer every month instead.